src/rosdiscover/models/move_base.py

```diff
@@ -1,4 +1,7 @@
 from ..interpreter import model
+from loguru import logger
```

Comment:
Apologies for the nitpicking comment: The general order of imports should be Python standard library stuff in one block at the top, followed by external imports in another block (e.g., `loguru`, and then, finally, relative imports in a last block: https://www.python.org/dev/peps/pep-0008/#id23

The linter doesn't seem to be clever enough to figure this out.

Reply:
Think done.